datasets/new_york/pipelines/_images/run_csv_transform_kub_nypd_mv_collisions/csv_transform.py

```diff
@@ -0,0 +1,317 @@
+# Copyright 2021 Google LLC
```

Comment:
2022 ?

Reply:
@happyhuman We are no longer required to change this year value.  It will be done automatically.  See @adlersantos